src/exceptions/fieldDoesNotExist.exeception.ts

```diff
@@ -0,0 +1,7 @@
+export class FieldDoesNotExist extends Error {
```

Comment:
The class name should actually be `FieldDoesNotExistException`.
Please also mind the fact that there is a typo in the file name. It should be `fieldDoesNotExist.exception.ts`, mind the `.exception` part.